conans/client/generators/boostbuild.py

```diff
@@ -0,0 +1,50 @@
+
+"""
+Boost Build Conan Generator
+
+This is a simple project-root.jam generator declaring all conan dependencies
+as boost-build lib targets. This lets you link against them in your Jamfile
+as a <library> property. Link against the "conant-deps" target.
+
+"""
+
+from conans.model import Generator
+from conans import ConanFile
+
+def JamfileOutput(dep_cpp_info):
+    out = ''
+    for lib in dep_cpp_info.libs:
+        out += 'lib %s :\n' % lib
+        out += '\t: # requirements\n'
+        out += '\t<name>%s\n' % lib
+        out += ''.join('\t<search>%s\n' % x for x in dep_cpp_info.lib_paths)
```

Comment:
For Windows paths, the \\ should be scaped, use:
`from conans.client.files import unix_path`
`unix_path(x) for x in dep_cpp_info.lib_paths`
Apply the same for include_paths

Reply:
Beware that ``unix_path`` might not work in all contexts, I don't think it is the good solution here

Reply:
Then we should `x.replace("\\", "/") for x in dep_cpp_info.lib_paths`

Reply:
the path conan passes in, is it always a windows path? Windows paths do work in boost-build on windows, but the backslashes need to be escaped. I would have expected the path from conan to be in the native format, I'm not 100% sure unix paths always work on windows here.

A related issue, if there are spaces in the path, there needs to be quotes around it. It probably makes sense to always quote these paths.